inspirehep/dojson/hep/receivers.py

```diff
@@ -22,40 +22,32 @@
 
 """Contains signal receivers for HEP dojson processing."""
 
+from __future__ import absolute_import, division, print_function
+
+from itertools import chain
+
 from invenio_records.signals import before_record_insert, before_record_update
 
 from inspirehep.utils.date import create_earliest_date
+from inspirehep.utils.helpers import force_force_list
+from inspirehep.utils.record import get_value
 
 
 @before_record_insert.connect
 @before_record_update.connect
 def earliest_date(sender, *args, **kwargs):
```

Comment:
Uh! I now realize this should not be stored in the DB, but only sent to ES. The earliest date is something that should be dynamically computed after all other related metadata dates.